### PR TITLE
[UI/About] Remove redundant fstream.close, move license loading

### PR DESCRIPTION
**Pull request by VPeruS**

### CODE REVIEW

File: UI/About.cpp [hunk 1]
```diff
@@ -53,6 +48,20 @@ void About::CompleteConstruction() {
         boost::bind(&About::ShowVision, this));
 }
 
+void About::LoadLicense(const boost::filesystem::path& licensePath)
+{
```
Comment (geoffthemedio) [inline UI/About.cpp]: move `{` to end of previous line

File: UI/About.cpp [hunk 2]
```diff
@@ -53,6 +48,20 @@ void About::CompleteConstruction() {
         boost::bind(&About::ShowVision, this));
 }
 
+void About::LoadLicense(const boost::filesystem::path& licensePath)
```
Comment (geoffthemedio) [inline UI/About.cpp]: use `variable_names_with_underscores`

**Comment by geoffthemedio** [comment]
TravisCI SmokeTestGame failure doesn't seem relevant to these changes.

File: UI/About.h [hunk 3]
```diff
@@ -24,6 +27,7 @@ class About : public CUIWnd {
 
 private:
     void DoLayout();
+    void LoadLicense(const boost::filesystem::path& licensePath);
```
Comment (geoffthemedio) [inline UI/About.h]: probably doesn't need to be in the header... just in an anonymous namespace in the .cpp

File: UI/About.cpp (hunk 2 above)
Comment (VPeruS) [reply UI/About.cpp]: done

File: UI/About.cpp (hunk 1 above)
Comment (VPeruS) [reply UI/About.cpp]: done

File: UI/About.h (hunk 3 above)
Comment (VPeruS) [reply UI/About.h]: moved to anon. ReadFile

**Comment by Vezzra** [comment]
Ok to merge?

**Comment by geoffthemedio** [comment]
looks fine

**Comment by Vezzra** [comment]
@VPeruS, do you want to provide a real name for the credits?

**Comment by VPeruS** [comment]
@Vezzra vperus is fine

**Pull request merged**